cds_ils/importer/providers/cds/rules/journal.py

```diff
@@ -85,11 +85,19 @@ def internal_notes(self, key, value):
     return {"value": clean_val("a", value, str, req=True)}
 
 
-@model.over("note", "^935__")
+@model.over("note", "(^935__)|(^866__)")
 @out_strip
 def note(self, key, value):
     """Translates note field."""
-    return clean_val("a", value, str, req=True)
+    notes_list = [self.get("note", "")]
+    if key == "935__":
+        notes_list.append(clean_val("a", value, str, req=True))
```

Comment:
why is the subfield a required?

Reply:
The code was like this before. I removed it now since in the schema it is not required.